content/post/proxysql-webinar-2023.02.01/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: MariaDB and Percona-XtraDB Support for KubeDB ProxySQL, ProxySQL Dashboard & Alerts
+date: "2023-02-01"
+weight: 20
+authors:
+- Tasdidur Rahman
+tags:
+- kubedb
+- kubernetes
+- mysql
+- mariadb
+- percona-xtradb
+- provisioner
+- proxysql
+---
+
+## Summary
+
+On FEB 1st, 2023, AppsCode held a webinar on **"MariaDB and Percona-XtraDB Support for KubeDB ProxySQL, ProxySQL Dashboard & Alerts"**. <br>
+The essential contents fo this webinar are: <br>
+* MariaDB and Percona-XtraDB backend support for KubeDB ProxySQL
+* Grafana dashboard for KubeDB ProxySQL
+* Prometheus alerts for KubeDB ProxySQL
+* Q&A session
+
+
+## Description of the Webinar
+
+The speaker started the webinar mentioning the updated backend support that have been recently integrated to KubeDB ProxySQL. Starting with the new MariaDB backend support demonstration, the speaker ended up showing the Percona-XtraDB backend support with the user synchronization and TLS.
```

Comment:
The webinar started by mentioning the updated backend support that has been recently integrated to KubeDB ProxySQL. Starting with the new MariaDB backend support demonstration, and ended up showing the Percona-XtraDB backend support with the user synchronization and TLS.

Reply:
done